README.md

```diff
@@ -13,6 +13,10 @@ Install the required ceph packages for ansible on the controller, on Debian that
 ```
 apt install --no-install-recommends ceph-base ceph-common
 ```
+Version required : reef
+
+Install instructions : [Ceph Doc](https://docs.ceph.com/en/latest/install/get-packages/)
+
```

Comment:
As mentioned, that's not actually required, I've been doing testing using 16.2 (stock from Debian 12) and heard it working even with 15.2 (Ubuntu 20.04).